demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -0,0 +1,841 @@
+
+        elif file_type == 'dashboard':
+            dashboard_validator = DashboardValidator(structure_validator, ignored_errors=pack_error_ignore_list,
+                                                     print_as_warnings=self.print_ignored_errors)
+            return dashboard_validator.is_valid_dashboard(validate_rn=False)
+
+        elif file_type == 'incidenttype':
+            incident_type_validator = IncidentTypeValidator(structure_validator, ignored_errors=pack_error_ignore_list,
+                                                            print_as_warnings=self.print_ignored_errors)
+            if is_modified and self.is_backward_check:
+                return all([incident_type_validator.is_valid_incident_type(validate_rn=False),
+                            incident_type_validator.is_backward_compatible()])
+            else:
+                return incident_type_validator.is_valid_incident_type(validate_rn=False)
+
+        elif file_type == 'mapper':
+            mapper_validator = MapperValidator(structure_validator, ignored_errors=pack_error_ignore_list,
+                                               print_as_warnings=self.print_ignored_errors)
+            return mapper_validator.is_valid_mapper(validate_rn=False)
+
+        elif file_type == 'classifier_5_9_9':
+            classifier_validator = ClassifierValidator(structure_validator, new_classifier_version=False,
+                                                       ignored_errors=pack_error_ignore_list,
+                                                       print_as_warnings=self.print_ignored_errors)
+            return classifier_validator.is_valid_classifier(validate_rn=False)
+
+        elif file_type == 'classifier':
+            classifier_validator = ClassifierValidator(structure_validator, ignored_errors=pack_error_ignore_list,
+                                                       print_as_warnings=self.print_ignored_errors)
+            return classifier_validator.is_valid_classifier(validate_rn=False)
+
+        elif file_type == 'widget':
+            widget_validator = WidgetValidator(structure_validator, ignored_errors=pack_error_ignore_list,
+                                               print_as_warnings=self.print_ignored_errors)
+            return widget_validator.is_valid_file(validate_rn=False)
+
+        else:
+            error_message, error_code = Errors.file_type_not_supported()
+            if self.handle_error(error_message=error_message, error_code=error_code, file_path=file_path):
+                return False
+
+        return True
+
+    def run_validation_using_git(self):
+        """Runs validation on only changed packs/files (g)
+        """
+        self.branch_name = self.get_current_working_branch()
+        if self.branch_name != 'master' and (not self.branch_name.startswith('19.') and
+                                             not self.branch_name.startswith('20.')):
+            compare_type = '.'
```

Comment:
what does that mean?

Reply:
![image](https://user-images.githubusercontent.com/50295826/85220099-a100f800-b3b1-11ea-9363-2398a1dc69ab.png)
from: https://stackoverflow.com/questions/7251477/what-are-the-differences-between-double-dot-and-triple-dot-in-git-dif/46345364

If I am on a non-master branch - I want the comparison to be to the master origin.
If I am on a master or new release branch - I want to compare to a release branch and not the master origin.

I'll add a comment explaining this, and make it a bit more explicit.